test/command/9652.md

```diff
@@ -0,0 +1,9 @@
+````
+% pandoc -f markdown -t html --embed-resources
+```{=html}
+<img class="something" src="9652.svg" />
+```
+^D
+<svg id="svg_b627f92299158b36552b" role="img" class="something" width="504.00pt" height="360.00pt" viewBox="0 0 504.00 360.00" class="something please-do-not-delete-me">
+</svg>
```

Comment:
This doesn't seem quite right; note that there are two `class` attributes.

Reply:
Oh that's unfortunate. I completely missed that, thank you.